docs/source/reference/urls.md

```diff
@@ -123,6 +123,8 @@ Handling this URL is the most complicated condition in JupyterHub,
 because there can be many states:
 
 1. server is not active
+NOTE: These next two don't render properly in "EPUB" format when downloaded
```

Comment:
I would go ahead and remove them since they are confusing as written. 

Perhaps something like: 

Handling this URL depends on two conditions: whether a requested user is found as a match; the state of the requested user's notebook server.